source/server/options_impl.cc

```diff
@@ -81,6 +81,8 @@ OptionsImpl::OptionsImpl(std::vector<std::string> args,
   TCLAP::SwitchArg reject_unknown_dynamic_fields("", "reject-unknown-dynamic-fields",
                                                  "reject unknown fields in dynamic configuration",
                                                  cmd, false);
+  TCLAP::SwitchArg ignore_unknown_dynamic_fields("", "ignore-unknown-dynamic-fields",
+                                                 "GENERA more words.", cmd, false);
```

Comment:
?

Reply:
I need to port my note-to-self presubmit to git, clearly ;)

Reply:
Yeah, there are still a few of these in the PR, searching for "genera " in the diff.

Reply:
I think I excised the last one (at least, grep -r thinks so too). I have not yet managed to make a local pre-commit that checks for them, but hey, something to hack on tomorrow.